raft.go

```diff
@@ -620,7 +620,7 @@ func (r *Raft) run() {
 // runFollower runs the FSM for a follower.
 func (r *Raft) runFollower() {
 	didWarn := false
-	r.logger.Printf("[INFO] raft: %v entering Follower state", r)
+	r.logger.Printf(`[INFO] raft: %v entering Follower state (Leader: "%s")`, r, r.Leader())
```

Comment:
Fun fact (you don't need to change this at all, but in case you didn't know), you can use the `%q` syntax to get a quoted string. This will also call `String()` on it if it is a `Stringer` impl.